base_layer/core/src/validation/block_validators.rs

```diff
@@ -190,7 +193,7 @@ impl StatelessValidation<Block> for MockStatelessBlockValidator {
     /// 1. Where all the rules for the spent outputs followed?
     /// 1. Was cut through applied in the block?
     fn validate(&self, block: &Block) -> Result<(), ValidationError> {
-        check_coinbase_output(block, &self.rules.consensus_constants())?;
+        check_coinbase_output_dont_check_amount(block, self.rules.clone(), &self.factories)?;
```

Comment:
You can just use the usual  check_coinbase_output fn here. Don't see why you're creating a new version

Reply:
check comment above